hasura/migrations/default/1713898610628_alter_table_public_app_metadata_miniapps_add_support_info/up.sql

```diff
@@ -1,6 +1,5 @@
 
-alter table "public"."app_metadata" add column "support_email" varchar
- null;
+alter table "public"."app_metadata" add column "support_email" varchar NOT NULL DEFAULT '',
```

Comment:
why `not null` with default empty string instead of null?

Reply:
Since with varchar you get two nullable values. Null and empty string

Reply:
Yes, but on the code side, it will be not accurate, because when you generate types support_email field will be always a string type which will cause the value existence check missing.